platformcomponents/desktop/attachment-file-preview-thumbnail.json

```diff
@@ -0,0 +1,12 @@
+{
+  "attachment-file-preview-thumbnail": {
+    "sent": {
+      "comment": "Applied to file attachments",
+      "figma": "https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows?node-id=29250%3A45485",
+      "border": "@theme-outline-secondary-normal",
+      "icon": "@theme-common-text-white",
```

Comment:
In the rebrand, the icon is always white, because of the accent background color, capture here under key `icon-background`. For now it's blue, but I there will be various other colors in the future based on the file type.

Reply:
Attachment file preview thumbnail is this one.
<img width="379" alt="image" src="https://user-images.githubusercontent.com/56999622/161941113-cade9870-8fdd-4a95-8258-ee68b2b9bff3.png">
It does not have a white icon with accent background. It has a black icon (theme-common-text-black) with a white background (common-button-primary-normal).

Reply:
This OTHER one here, with a white icon and accent background, is a different component, called `attachment file default`, which you added as well in this PR.

<img width="387" alt="image" src="https://user-images.githubusercontent.com/56999622/161941399-2854aff1-4609-4f90-9d7d-f1d7a416ea22.png">

Reply:
So, if I read this right, the initial issue was that for `attachment-file-preview-thumbnail` I was adding colors for icon, which doesn't exist for this component.
This has been addressed.
Changing the conversation to Resolved.